kernelci/cli/show.py

```diff
@@ -56,6 +56,13 @@ def _dump_artifacts(self, artifacts):
             print(f"  {self._color(name, 'blue')}")
             print(f"    {url}")
 
+    def _dump_data(self, data):
+        max_len = max(len(key) for key in data)
```

Comment:
If data is iterable this can be a better use of `max`

```suggestion
        max_len = max(data, key=len)
```

Reply:
This actually returns the longest key, so I would need to do `len(max(data, key=len))` which I think seems more cryptic than just a comprehension with `len(key)`.